library/core/talaiot/src/main/kotlin/com/cdsap/talaiot/metrics/GradleScanLinkMetric.kt

```diff
@@ -24,39 +24,6 @@ class GradleScanLinkMetric : BuildResultMetric<String?>(
         val services = gradleInternal.services
 
         when {
-            GradleVersion.current() > GradleVersion.version("5.0")
```

Comment:
So we don't provide a way to use the new plugin with the old versions of gradle? This sounds just like the approach a lot of plugins are choosing and they use the same version string for the plugin as gradle, e.g. gradle 6.7 requires plugin version 6.7

Reply:
We don't provide support for Gradle < 6.  

Sounds wrong, but the current build system's state and the main components of the build stack(agp/kgp) force you to be updated and prepare for the next features. 
Talking about AS/AGP we have:
AGP 4.0.0+ -- Gradle 6.1.1+
AGP 4.1.0+ -- Gradle 6.5+
If one project wants to build with AS 4.1, use agp new features or build improvements, it's forced to update Gradle 6.5. 
Google is doing a great job creating hype with Compose and forcing the teams to talk about build possibilities to integrate compose, translated in embrace newer build tools.

That's why in the short-mid term, I'm focusing in be compatible with Gradle 7 and the Configuration Cache feature(Google is doing a great job creating hype with Compose and forcing the teams to talk about build possibilities to integrate compose, agp, kgp). 

I will add a couple of E2E tests and put ready for review

Reply:
That branch condition also include [6.0, 6.7), I'm kinda interested in those versions as well as I expect they're still used

Reply:
added back support but from 6.0.
Added e2e tests, ready for review